### PR TITLE
Event slide

### PR DESCRIPTION
**Pull request by lasseborly**
This adds the event slide to the bundle along with a couple of new tools.

There are some terrible nested ternaries in there but because of the architecture it's hard to figure out how you would go about sharing computed valued between the tools and the actual slide.

### CODE REVIEW

File: Resources/public/templates/ding2/slides/event/event-edit.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <div
+      class="event--image"
+      data-ng-style="{'background-image': 'url(' + ikSlide.currentImage + ')'}"
+    ></div>
+    <div
+      class="event--info-header"
+      spellcheck="false"
+      data-ng-style="{'border-color': ikSlide.options.headercolor}"
+    >{{ ikSlide.options.date.text ? ikSlide.options.date.text : ikSlide.options.date.to
+      && ikSlide.options.date.to ? 'Dato: ' + ikSlide.options.date.from + ' - ' +
```
Comment (danquah) [inline Resources/public/templates/ding2/slides/event/event-edit.html]: How about handling this in a js function in the controller? The slideshow slide adds a functions to ng's scope and calls it from its template - maybe you could do the same to get rid of the nested ternary?

https://github.com/os2display/default-template-bundle/blob/master/Resources/public/apps/toolsModule/slideshow-order-editor.js#L24

https://github.com/os2display/default-template-bundle/blob/master/Resources/public/apps/toolsModule/slideshow-order-editor.html#L25

File: Resources/public/templates/ding2/slides/event/event.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  "orientation": "portrait",
+  "ideal_dimensions": {
+    "width": "1080",
+    "height": "1920"
+  },
+  "empty_options": {
+    "fontsize": 60,
+    "responsive_fontsize": 4,
+    "bgcolor": "#fff",
+    "headercolor": "#4e4a47",
```
Comment (danquah) [inline Resources/public/templates/ding2/slides/event/event.json:25]: Would be nice to have this in an external configuration-file - but I'm cool with figuring out how to do that in a future slide.

**Review by danquah — APPROVED** [review]
Had an suggestion for how we might handle the nested ternaries, but I'm also good with merging. Good stuff!

File: Resources/public/templates/ding2/slides/event/event-edit.html (hunk 1 above)
Comment (lasseborly) [reply Resources/public/templates/ding2/slides/event/event-edit.html]: :+1:

Comment (lasseborly) [reply Resources/public/templates/ding2/slides/event/event-edit.html]: The ternary has been vanquished from the kingdom.

File: Resources/public/templates/ding2/slides/event/event.json (hunk 2 above)
Comment (lasseborly) [reply Resources/public/templates/ding2/slides/event/event.json:25]: Wouldn't that be a core change needed?

**Pull request merged**